plugins/woocommerce/includes/tracks/events/class-wc-products-tracking.php

```diff
@@ -330,6 +331,13 @@ protected function get_product_screen( $hook ) {
 		) {
 			return 'edit';
 		}
+
+		if(
+			'product_page_product_importer' === $hook
+		) {
```

Comment:
This seems to break some lint errors

Reply:
Whoops, still need to get my phpcs thing working.